emitter.ts

```diff
@@ -288,7 +292,11 @@ export async function emitter(
 
         if (event == null) {
           logger.error(() =>
-            `Event does not exist in DB, blockNumber-txIndex-logIndex: ${x.blockTimestamp}-${x.txIndex}-${x.logIndex}  blockHash: ${x.blockHash}  address: ${x.address}  event signature hash: ${x.sigHash}  topics: ${
+            `Event does not exist in DB, blockNumber-txIndex-logIndex: ${x.blockTimestamp}-${x.txIndex}-${x.logIndex}  blockHash: ${
+              toHex(x.blockHash)
+            }  address: ${toHex(x.address)}  event signature hash: ${
+              toHex(x.sigHash)
+            }  topics: ${
               x.topics.map((topic, i) => `[${i}] ${toHex(topic)}`).join(" ")
```

Comment:
Missed from previous PR, topic seems to be nullish checked before calling toHex, also.

(+ [${i + 1}] maybe?)

Reply:
Amended!